src/layer-composer/types.ts

```diff
@@ -12,7 +12,9 @@ export enum Group {
   Basemap = 'basemap', // Satellite tiles
   Heatmap = 'heatmap', // Fill/gradient-based heatmaps
   BasemapFill = 'basemapFill', // Landmass
-  OutlinePolygons = 'outlinePolygons', // Conbtext layers with an outlined/hollow style such as EEZ, RFMOs, etc
+  BasemapPolygons = 'basemapPolygons', // Polygons  that need to be rendered below landmass
```

Comment:
Seems confusing to me that the CartoPolygons generator would generate layer in a `BasemapPolygons` group (while a BasemapGenerator exist)
Maybe 
~BasemapPolygons~ -> `OutlinePolygonsBackground` ?